<?xml version="1.0"?>
<launch>
  <include file="$(find tsim)/launch/turtle_pent.launch" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find nuturtle_description)/config/config.rviz" />

  <node pkg="rigid2d" type="odometer" name="odometer" output="screen">
    <remap from="joint_state" to="joint_states"/>
    <param name="~body_frame_id" type="str" value="base_link" />
    <param name="~odom_frame_id" type="str" value="odom" />
  </node>


   <param name="robot_description" command="xacro '$(find nuturtle_description)/urdf/diff_drive.urdf.xacro'" />

  <rosparam command="load" file="$(find nuturtle_description)/config/diff_params.yaml" />

  <node pkg="rigid2d" type="fake_diff_encoders" name="fake_diff_encoders"  output="screen">
  </node>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
  <param name="height" value="5" />
  <param name="trans_vel" value="2" />

</launch>
